tensorflow/tools/ci_build/linux/cpu/run_py2_core.sh

```diff
@@ -32,7 +32,7 @@ export PYTHON_BIN_PATH=`which python2`
 yes "" | $PYTHON_BIN_PATH configure.py
 
 # Run bazel test command. Double test timeouts to avoid flakes.
-bazel test --test_tag_filters=-no_oss,-oss_serial,-gpu,-benchmark-test --test_lang_filters=py -k \
+bazel test --test_tag_filters=-no_oss,-no_oss_py2,-oss_serial,-gpu,-benchmark-test --test_lang_filters=py -k \
```

Comment:
Do we have these tags in our BUILD files anywhere?
@hyeygit to make sure she is aware of the new tag.,

Reply:
From https://source.cloud.google.com/results/invocations/60743e4c-b730-40fe-a5bd-6f2067c2dbba/details   I can see -no_oss_py2 in "invocation details".

Reply:
I was not aware of this tag being added. LGTM.